tensorflow/python/BUILD

```diff
@@ -3918,6 +3918,7 @@ cuda_py_test(
     srcs = [
         "grappler/layout_optimizer_test.py",
     ],
+    tags = ["manual"],
```

Comment:
This needs to be moved to below addtional_deps to compliant to buildifier order.